src/highdicom/valuerep.py

```diff
@@ -0,0 +1,68 @@
+from math import log10, floor
+
+from numpy import isfinite
+
+
+def get_ds_string(f: float) -> str:
```

Comment:
Wouldn't it make sense for this to be in pydicom instead?

Reply:
@pieper I agree, this would ideally go into pydicom. There is an option issue (see https://github.com/pydicom/pydicom/issues/1264). @CPBridge have you considered creating a PR to update `pydicom.valuerep.DS`?

Reply:
@CPBridge given that this has been merged into pydicom, can we remove this module?

Reply:
yes, there's not sense reviewing this PR in it's current form